.github/workflows/main.yml

```
@@ -22,71 +23,76 @@
   Linting:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v4 # Recommended to use latest stable v4 or v3
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)